std-bits/base/src/main/java/org/enso/base/enso_cloud/EnsoSecretHelper.java

```diff
@@ -208,4 +215,43 @@ public static void simulateEnsoSecretReaderReload() {
   private static final Comparator<Pair<String, String>> headerNameComparator =
       Comparator.comparing((Pair<String, String> pair) -> pair.getLeft())
           .thenComparing(Comparator.comparing(pair -> pair.getRight()));
+
+  private static InputStream decodeContentEncoding(
+      InputStream stream, HttpHeaders headers) {
+    var encOpt = headers.firstValue("content-encoding");
+    if (!encOpt.isEmpty()) {
+      String encoding = encOpt.get().toLowerCase();
```

Comment:
could simplify with something like:
```suggestion
    String encoding = headers.firstValue("content-encoding").orElse("");
```

Reply:
Done